system-metrics/README.md

```diff
@@ -0,0 +1,20 @@
+## system-metrics
+
+This function exposes metrics from Prometheus.
+
+It queries Prometheus for a function's 2xx and non-2xx invocations count and returns a JSON response of type:
+
+```json
+{
+    "success": 10,
+    "failure": 8
+}
+```
+
+It takes function's name (f.e. `myFunction`) from a query and is invoked by GET request to
+http://gateway-url:8080/function/system-metrics?function=myFunction
```

Comment:
Does it only work on query-string or HTTP path also?

Reply:
For the moment it doesn't work with a path. Does this need an update?

Reply:
I think it's ok for now